charts/scalardb-analytics-postgresql/templates/scalardb-analytics-postgresql/deployment.yaml

```diff
@@ -0,0 +1,154 @@
+        {{- if .Values.scalardbAnalyticsPostgreSQL.podAnnotations }}
+        {{- toYaml .Values.scalardbAnalyticsPostgreSQL.podAnnotations | nindent 8 }}
+        {{- end }}
+      labels:
+        {{- include "scalardb-analytics-postgresql.selectorLabels" . | nindent 8 }}
+    spec:
+      restartPolicy: Always
+      {{- if .Values.scalardbAnalyticsPostgreSQL.serviceAccount.serviceAccountName }}
+      serviceAccountName: {{ .Values.scalardbAnalyticsPostgreSQL.serviceAccount.serviceAccountName }}
+      {{- end }}
+      automountServiceAccountToken: {{ .Values.scalardbAnalyticsPostgreSQL.serviceAccount.automountServiceAccountToken }}
+      terminationGracePeriodSeconds: 60
+      {{- with .Values.scalardbAnalyticsPostgreSQL.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.scalardbAnalyticsPostgreSQL.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.scalardbAnalyticsPostgreSQL.securityContext | nindent 12 }}
+          image: "{{ .Values.scalardbAnalyticsPostgreSQL.image.repository }}:{{ .Values.scalardbAnalyticsPostgreSQL.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.scalardbAnalyticsPostgreSQL.image.pullPolicy }}
+          ports:
+            - containerPort: 5432
+          env:
+          - name: POSTGRES_DB
+            value: "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.databaseName }}"
+          envFrom:
+          - secretRef:
+              name:  "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.secretName }}"
+          {{- if .Values.scalardbAnalyticsPostgreSQL.secretName }}
+          - secretRef:
+              name: "{{ .Values.scalardbAnalyticsPostgreSQL.secretName }}"
+          {{- end }}
+          volumeMounts:
+            - mountPath: /etc/database.properties
+              name: database-properties-volume
+              subPath: database.properties
+          resources:
+            {{- toYaml .Values.scalardbAnalyticsPostgreSQL.resources | nindent 12 }}
+          startupProbe:
+            exec:
+              command:
+                - psql
+                - -d
+                - "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.databaseName }}"
+                - -U
+                - postgres
+                - -c
+                - "SELECT 1"
+            failureThreshold: 60
+            periodSeconds: 5
+          livenessProbe:
+            exec:
+              command:
+                - psql
+                - -d
+                - "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.databaseName }}"
+                - -U
+                - postgres
+                - -c
+                - "SELECT 1"
+            failureThreshold: 3
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+        - name: schema-importer
```

Comment:
As I mentioned in the PR description, this chart run `Schema Importer` as a sidecar.